static/styles/rendered_markdown.css

```diff
@@ -309,8 +309,15 @@
         margin-bottom: 5px;
         margin-left: 5px;
         height: 100px;
-        display: block !important;
+        width: 150px;
+        display: inline-block !important;
         border: none !important;
+
+        a {
```

Comment:
This is... probably not needed! My original idea was that we'd need to have this take up the parent explicitly; but I realize this is not needed for an inline anchor element.